README.md

```diff
@@ -1,6 +1,17 @@
 # The Yorick meta-tracer.
 
-[ykrustc](https://github.com/softdevteam/ykrustc) is a fork of the Rust
-language which aims to build a meta-tracing system from Rust. The tracing
-system as a whole is called Yorick. Imagine PyPy, but where your interpreter is
-written in Rust instead of RPython.
+Yorick is a fork of the Rust language which aims to build a meta-tracing system
+from Rust. Imagine PyPy, but where your interpreter is written in Rust instead
+of RPython.
+
+This repository houses the non-compiler components of the system.
+
+The compiler repo can be found [here](https://github.com/softdevteam/ykrustc).
+
+## Contributors
+
+Yorick is developed by the Software Development Team at King's College London,
```

Comment:
I think we can be more inclusive here. Maybe something like "Yorick is developed by a team of people which can be seen in the <graph of contributors>." I don't think we need a special mention.

Reply:
Fixed in 12d05fc4eb8d7f82ba13816fca869faab8752963